portfolio/src/main/webapp/Network_Info.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Web Information</title>
+	<link rel="stylesheet" href="Network_Info.css">
+	<link rel="stylesheet" href="General.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="network_body">
+		<div class="drop_menu">
+			<button class="menu_button">Page Menu</button>
+			<div class="dropdown_content">
+				<a href="Home.html">Home</a>
+				<a href="Image_Gallery.html">Images</a>
+				<a href="Network_Info.html">Network Info</a>
+			</div>
+		</div>
+		<a href="Home.html" class="padding previous round">&#8249;</a>
+		<h1>
+			<center>Networking Info</center>
+		</h1>
+		<h2>LinkedIn</h2>
+		<p>
+			<a href="https://www.linkedin.com/in/charles-floeder-878a12192/2">
+				LinkedIn URL
+			</a>
+		</p>
+		<h2>GitHub</h2>
+		<p>username: cxfloeder</p>
```

Comment:
You might want to link to your GitHub page. That will make it easier for users to find the information they are looking for.

Reply:
fixed